clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10647,8 +10647,11 @@ static void getNonTripleBasedSYCLPostLinkOpts(const ToolChain &TC,
                      options::OPT_fsycl_esimd_force_stateless_mem, false))
     addArgs(PostLinkArgs, TCArgs, {"-lower-esimd-force-stateless-mem=false"});
 
+  // TODO Enable when SYCL RT supports dynamic linking
```

Comment:
It will be cleaner and also avoid possible merge conflicts to remove this block of code and reintroduce when support is ready. Tracking can be done elsewhere. Thanks

Reply:
Code removed.